lib/modules/manager/gradle/extract.ts

```diff
@@ -70,6 +80,15 @@ export async function extractAllPackageFiles(
       } else if (isTOMLFile(packageFile)) {
         const updatesFromCatalog = parseCatalog(packageFile, content);
         extractedDeps.push(...updatesFromCatalog);
+      } else if (isGcvPropsFile(packageFile)) {
+        if (usesConsistentVersionPlugin && packageFile === VERSIONS_PROPS) {
```

Comment:
I guess the idea with `packageFile === VERSIONS_PROP` is to only run this on `versions.prop` files but not on `versions.lock` files?

In that case, let `isGcvPropsFile` only check on `VERSIONS_PROP` and not the lock file.

Reply:
Yes, I had that at first, but if `isGcvPropsFile()` returns false for the lock file, then the `versions.lock` will not be skipped, but parsed by the `else` clause, as if it was a `build.gradle` file. So I want to trigger this else clause for both files, but do NOP on the lock file while the props file triggers the parsing (of both).

Reply:
Ok, makes sense to catch them both there 👍

Reply:
I split them out so now I have one NOOP else for lockfile and then a simpler code flow in the else clause for props.